composer.json

```diff
@@ -26,7 +26,8 @@
         "illuminate/contracts": "~5.0",
         "squizlabs/php_codesniffer": "~1.5",
         "pagerfanta/pagerfanta": "~1.0.0",
-        "zendframework/zend-paginator":"~2.3"
+        "zendframework/zend-paginator":"~2.3",
+        "doctrine/orm": "^2.5"
```

Comment:
I realize that there's precedent here, because the other are required, but doctrine/orm is a huge dependency to pull in. I'd rather see it in the "suggested" array and people can require it if they like.